### PR TITLE
Updated Installation instructions for new Releases

### PR DESCRIPTION
**Pull request by pichardoJ**
As explained in https://github.com/dracula/jetbrains/issues/9

### CODE REVIEW

**Comment by zenorocha** [comment]
Can you review @joaoevangelista?

**Comment by joaoevangelista** [comment]
@pichardoJ would you mind add the path for windows ? it would be "C:/Users/<you>/.{NameOfIDE}/config/colors".

File: src/content/jetbrains/index.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 	<h4>Activating theme</h4>
     <ol>
-        <li>Create a folder called <code>colors</code> inside <code>~/Library/Preferences/<b>{NameOfIDE}</b></code>.</li>
+        <li>Create a folder called <code>colors</code> inside <code>~/<b>.{NameOfIDE}</b>/config/colors/</code>.</li>
         <li>Copy the <code>./jetbrains/Dracula.icls</code> file to the <code>~/Library/Preferences/<b>{NameOfIDE}</b>/colors</code> directory on your local machine.</li>
```
Comment (joaoevangelista) [inline src/content/jetbrains/index.html]: We need change this line too, so the user knows that it needs to be copied to the right folder

**Comment by pichardoJ** [comment]
@joaoevangelista Ok, will do the changes, I'm not a mac user could you provide the path for macOS

**Comment by pichardoJ** [comment]
Could do something like 

#### Activating Theme

**Note**

The user folder is as follows for the corresponding operative system

* Windows: `C:\Users\<you>\`
* Mac or Linux `/home/<you>/` or ~

- Create a folder named `colors` inside `{UserFolder}/.{NameOfIDE}/config/`
- Copy the file `/jetbrains/Dracula.icls` to the `{UserFolder}/*.{NameOfIDE}*/config/colors` folder

what do you think?

**Comment by joaoevangelista** [comment]
Me neither, I'm a Windows user, but your suggestion might address both Unix and Windows, Mac I think is the already present instructions

**Comment by joaoevangelista** [comment]
Just add the current instruction as Mac, then we are good to go

**Comment by pichardoJ** [comment]
@joaoevangelista Is it ok with the new commit?

File: src/content/jetbrains/index.html [hunk 2]
```diff
@@ -23,8 +24,17 @@
 
 	<h4>Activating theme</h4>
     <ol>
-        <li>Create a folder called <code>colors</code> inside <code>~/<b>.{NameOfIDE}</b>/config/colors/</code>.</li>
-        <li>Copy the <code>./jetbrains/Dracula.icls</code> file to the <code>~/Library/Preferences/<b>{NameOfIDE}</b>/colors</code> directory on your local machine.</li>
+        <li><b>Windows</b></li>
+        <ol>
+            <li>Create a folder called <code>colors</code> inside <code>C:/Users/<b>{you}</b>/.<b>{NameOfIDE}</b>/config/colors/</code>.</li>
```
Comment (joaoevangelista) [inline src/content/jetbrains/index.html]: Did you mean create `colors` folder inside the config folder?

**Comment by joaoevangelista** [comment]
Almost, just a typo I think happened when you put the creation of the folder inside a folder called colors instead of config

**Comment by pichardoJ** [comment]
Oh damn

**Comment by pichardoJ** [comment]
@joaoevangelista Is the new one ok?

**Comment by joaoevangelista** [comment]
Awesome! Thanks <3
@zenorocha This one is good to go.
Also if possible close #19 in favor of this one. since I was not aware of that PR until now (sorry about that)

**Comment by zenorocha** [comment]
Great, thank you!

**Pull request merged**